test/blackbox-tests/test-cases/pkg/conflicts.t

```diff
@@ -26,3 +26,108 @@ The solver should say no solution rather than just ignoring the conflict.
         foo.0.0.1: Package does not satisfy constraints of local package x
   [1]
 
+There could be more than one conflict and they can have versions constraints:
+
+  $ mkpkg foo2 0.0.1
+  $ mkpkg bar2 << EOF
+  > depends: [ "foo2" ]
+  > EOF
+
+  $ solve_project << EOF
+  > (lang dune 3.11)
+  > (generate_opam_files true)
+  > (package
+  >  (name x)
+  >  (allow_empty)
+  >  (conflicts (foo (< 0.2)) (foo2 (< 0.2)))
+  >  (depends bar bar2))
+  > EOF
+  Error: Unable to solve dependencies for the following lock directories:
+  Lock directory dune.lock:
+  Couldn't solve the package dependency formula.
+  Selected candidates: bar.0.0.1 bar2.0.0.1 x.dev
+  - foo -> (problem)
+      No usable implementations:
+        foo.0.0.1: Package does not satisfy constraints of local package x
+  - foo2 -> (problem)
+      No usable implementations:
+        foo2.0.0.1: Package does not satisfy constraints of local package x
+  [1]
+
+When conflicts are obtained from an opam file instead of a dune-project,
+the behaviour should be the same:
+
+  $ dune build x.opam
+  $ cat x.opam | sed -n '/conflicts/,/]/p'
```

Comment:
Nitpick, but useless use of `cat`.

(That said, I love the `sed` trick, that is really cool and useful)

Reply:
Haha right, thanks!